### PR TITLE
Creating a file for Athens, Greece

### LINKED ISSUE #39: Create marinakoritsidou hometown file
## Athens, Greece
- [x] Create a branch
- [x] Create a file
- [x] Add a commit
- [x] Create a Pull Request
- [ ] Have a conversation
- [ ] Get a :+1: 
- [ ] :ship: it (a.k.a. merge)

### PR DESCRIPTION
**Pull request by marinakoritsidou**
This PR resolves #39

### CODE REVIEW

File: marinakoritsidou-athens.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Athens, Greece
+
+### Places to eat
```
Comment (marinakoritsidou) [inline marinakoritsidou-athens.md:3]: @KamaleswaranS You should start training for the 10K!

File: marinakoritsidou-athens.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+### Fun things to do
```
Comment (claudiu-mera) [inline marinakoritsidou-athens.md:7]: @marinakoritsidou what do you recommend?

File: marinakoritsidou-athens.md (hunk 1 above)
Comment (tinaGitStudent) [inline marinakoritsidou-athens.md:3]: @marinakoritsidou I love the Greek Food! Can't wait to see your recommendations!

**Comment by marinakoritsidou** [comment]
This looks :+1:

**Pull request merged**